Jenkinsfile

```diff
@@ -29,6 +30,73 @@ pipeline {
                 sh 'build/run make -j\$(nproc) test-integration'
             }
         }
+
+        stage('Integration Tests SetUp') {
+            steps {
+                sh 'tests/scripts/saveImages.sh'
+                stash name: 'repo', useDefaultExcludes: false
+            }
+        }
+
+        stage('Integration Tests - parallel runs') {
+            environment {
+                KUBE_VERSION = "v1.6"
+            }
+            steps {
+                parallel(
+                    "GCE_K8s_v1.7" : {
+                        node("gce_vm"){
+                            script {
+                                try{
+                                    withEnv(["KUBE_VERSION=v1.7"]){
+                                        unstash 'repo'
+                                        deleteDir()
+                                        sh "docker load -i rookamd64.tar"
+                                        sh "docker load -i toolboxamd64.tar"
+                                        sh 'build/run make -j\$(nproc) build.common'
```

Comment:
why is a build needed here? should be a pre-req for test-integration in the makefile?